website/src/pages/pricing.tsx

```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
             </Helmet>
             <div className="pricing-page">
                 <ContentSection color="primary" className="hero-section text-center py-5">
-                    <h1>Sourcegraph pricing</h1>
-                    <h3>
-                        Free to start. No commitment.
-                        <br />
-                        Only pay for active users.
-                    </h3>
+                    <h2>Universal Code Search, for less than the cost of one bad line of code.</h2>
```

Comment:
This is a little cheesy :). I think the line below is more effective (i..e, drop the "for less than" part)